### PR TITLE
README: add info about debugging contracts

### PR DESCRIPTION
**Pull request by chadoh**
Steps for troubleshooting (calls to) contracts using `truffle console`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,78 @@ $ npm run start:dot
...
+   ```
+
+   Or anything else listed in [the web3 docs](https://web3js.readthedocs.io/en/v1.2.1/web3-eth-contract.html)
+
+7. Make changes to your contract code
+
+8. Re-compile & deploy your contract
+
+   ```
+   ???
```
Comment (chadoh) [inline README.md]: @topocount what commands do we want here?

Comment (ottodevs) [reply README.md]: We should just remove 7 and 8 steps. Or replace by something like 
> ask a solidity developer to look into it if you think there is an issue with the contract

**Comment by ottodevs** [comment]
> Steps for troubleshooting (calls to) contracts using `truffle console`

I think the title could be confusing, it is important to understand the difference between debugging frontend issues with contract calls and debugging contracts.

For the latter there are some really good approaches, use remix, tests, vs code debugging, truffle console is so good but not the only one, why would we want to stick to just one method? Each developer would have his own tricks, maybe this is better to fit a technical article on the blog commenting on the different options instead, but this is maybe another topic :)

**Comment by ottodevs** [comment]
I would just put this PR on hold until the mainnet release is completed, I don't think we should  extra-time on making technical tutorials right now, but I definitely think this is a really nice idea to have in the future, maybe in a more visible place than the root README.md (we already have more specialized MD documents for technical topics)

**Comment by chadoh** [comment]
Thanks for the input, @ottodevs. This is what I came up with while in a call with Kevin, for debugging the contract. Sounds like maybe there are better ways to do it, though. I'll close this without merging; I wanted to have it documented somewhere, and I don't mind having it in an unmerged PR for now.

**Pull request closed without merging**